### PR TITLE
Enable Dependabot

### PR DESCRIPTION
**Pull request by JackPGreen**
Fixes https://github.com/hazelcast/hazelcast-cpp-client/issues/1232

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
```
Comment (ihsandemir) [inline .github/dependabot.yml:3]: does this mean dependabot will only be triggered if change in github-actions?

Comment (JackPGreen) [reply .github/dependabot.yml:3]: Yes, it will only raise PRs if there are updateable GitHub Actions, checked weekly.

Comment (ihsandemir) [reply .github/dependabot.yml:3]: What do you mean by `updateable GitHub Actions`? I thought that dependabot would check the code base on periodic basis regardless of github actions.

Comment (JackPGreen) [reply .github/dependabot.yml:3]: > What do you mean by `updateable GitHub Actions`? I thought that dependabot would check the code base on periodic basis regardless of github actions.

It means the GitHub Actions dependencies declared in the codebase - e.g.:
https://github.com/hazelcast/hazelcast-cpp-client/blob/b2d7e9424dddb6f6cd55512167ba29eb2fe905cc/.github/workflows/nightly-windows.yml#L78

**Review by ihsandemir — APPROVED** [review]

**Pull request merged**